app/controllers/events/email.py

```diff
@@ -0,0 +1,53 @@
+from flask import Flask, redirect, flash, url_for, request, g, render_template
+from flask_mail import Mail, Message
+from peewee import fn
+from app.models.interest import Interest
+from app.models.user import User
+from app.models.event import Event
+from app.models.eventParticipant import EventParticipant
+from app.models.programEvent import ProgramEvent
+from app.models.emailTemplate import EmailTemplate
+from app.models.emailLog import EmailLog
+from app.controllers.main import main_bp
+from app.logic.emailHandler import getInterestedEmails, getParticipantEmails, emailHandler
+from app import app
+
+@main_bp.route('/retrieveEmailTemplate', methods=['GET'])
+def retrieveEmailTemplate():
+    # Couldn't find a way to do this with render_template
+    # TODO: find a better way?
+    templateInfo = {}
+    emailTemplates = EmailTemplate.select()
+
+    for index, template in enumerate(emailTemplates):
+        templateInfo[index] = {
+            'purpose': template.purpose,
+            'subject':template.subject,
+            'body': template.body}
+    return templateInfo
+
+@main_bp.route('/fetchEmailLogData/<eventId>', methods=['GET'])
+def fetchEmailLogData(eventId):
+    dateSent = recipients = None
+    # TODO: Bruteforce implementation. Need to figure out a way to get the most recent email log for an event
+    emailLogs = EmailLog.select().where(EmailLog.event==eventId).order_by(EmailLog.dateSent.asc())
+    for emailLog in emailLogs:
+        recipients = emailLog.recipientsCategory
+        dateSent = emailLog.dateSent.strftime('%m/%d/%Y')
+    return {'recipients':recipients, 'dateSent':dateSent}

```

Comment:
This is a very brute force way to get the most recent email log. I searched about how to get the most recent using peewee but wasn't successful. Any ideas on how to do this?

In the email log, each email is tied to an event. Since users can send multiple emails for a single event, in the modal where it says "last email was sent ...." we need to show the latest one.